firebase-firestore/src/main/java/com/google/firebase/firestore/local/SQLitePersistence.java

```diff
@@ -102,18 +100,38 @@ public void onCommit() {
         public void onRollback() {}
       };
 
+  private SQLiteDatabase db;
+  private boolean started;
+
+  public SQLitePersistence(
+      Context context,
+      String persistenceKey,
+      DatabaseId databaseId,
+      LocalSerializer serializer,
+      LruGarbageCollector.Params params) {
+    this(
+        context,
+        persistenceKey,
+        databaseId,
+        serializer,
+        StatsCollector.newNoOpStatsCollector(),
+        params);
+  }
+
   public SQLitePersistence(
       Context context,
       String persistenceKey,
       DatabaseId databaseId,
       LocalSerializer serializer,
+      StatsCollector statsProvider,
```

Comment:
Your class is `StatsCollector` but the variable is `statsProvider`. Elsewhere you called it `statsCollector` too. Make these consistent?